.gitignore

```diff
@@ -2,8 +2,6 @@
 *~
 # Ignore Vim swap files.
 .*.swp
-# macOS-specific excludes
```

Comment:
This needs to be added again (you broke it out in another PR).

Reply:
Done.